contents/blog/posthog-vs-hotjar.md

```diff
@@ -0,0 +1,537 @@
+        <tr>
+			<td className="w-3/12"></td>
+            <td><strong>Hotjar</strong></td>
+            <td><strong>PostHog</strong></td>
+            <td></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>Click and tap heatmaps</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Show where users are clicking.</td>
+        </tr>
+        <tr>
+            <td><strong>Move heatmaps</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Show where users have moved their mouse on the screen.</td>
+        </tr>
+        <tr>
+            <td><strong>Scroll heatmaps</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Show how far down the page your users scroll.</td>
+        </tr>
+        <tr>
+            <td><strong>Rage clicks</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Show where users repeatedly click in a short period of time.</td>
+        </tr>
+        <tr>
+            <td><strong>Combine rage click data with product analytics</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>View related events and funnel data for users who rage click.</td>
+        </tr>
+        <tr>
+            <td><strong>Heatmap data filters</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Filter heatmaps by date, country etc.</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+### In-app messages and user research
+
+Hotjar enables you to hear from users with feedback widgets, surveys, and an integrated user research tool. PostHog also supports [feedback widgets](/apps/feedback-widget) and [user research scheduling](/apps/user-interviews), although not surveys. However, PostHog offers additional ways to communicate with your users, such as a [notification bar](/apps/notification-bar) and interactive [pop-ups](tutorials/react-popups) through the use of [feature flag payloads](/docs/feature-flags/payloads) and [site apps](/tutorials/build-site-app). Also, since PostHog provides product analytics, you can easily define your target users who should see your widgets.
```

Comment:
```suggestion
Hotjar includes feedback widgets, surveys, and an integrated user research tool for gathering feedback from users. PostHog also supports [feedback widgets](/apps/feedback-widget) and [user research scheduling](/apps/user-interviews), although not surveys.

PostHog also offers additional ways to communicate with your users, such as a [notification bar](/apps/notification-bar) and interactive [pop-ups](tutorials/react-popups) through the use of [feature flag payloads](/docs/feature-flags/payloads), and [site apps](/tutorials/build-site-app). Also, since PostHog provides product analytics, you can easily define your target users who should see your widgets.
```